tests/validation/cattlevalidationtest/core/test_services.py

```diff
@@ -1337,6 +1337,39 @@ def test_service_retain_ip(super_client, client):
     assert externalId != new_externalId
 
 
+def test_services_rolling_strategy(super_client, client,
+                                   socat_containers):
+    launch_config = {"imageUuid": SSH_IMAGE_UUID,
+                     }
+    service, env = create_env_and_svc(client, launch_config, 2)
+
+    env = env.activateservices()
+    service = client.wait_success(service, 300)
+    container_list = get_service_container_list(super_client, service)
+    assert len(container_list) == 2
+    for con in container_list:
+        assert con.state == "running"
+        assert con.startCount == 1
+
+    # Specify rolling restart strategy with batchsize 2 and interval of 1000 ms
+    rollingrestartstrategy = {"batchsize": 2,
+                              "intervalMillis": 1000
+                              }
```

Comment:
Can you increase the scale to 5 , so that restarts are attempted in multiple batches?